crates/core/tedge_mapper/src/az/mapper.rs

```diff
@@ -57,3 +72,27 @@ fn get_topic_filter(tedge_config: &TEdgeConfig) -> TopicFilter {
     }
     topics
 }
+
+fn built_in_bridge_rules(remote_clientid: &str) -> anyhow::Result<BridgeConfig> {
```

Comment:
Same comment as for the AWS bridge. It would be good to avoid repeating ourselves as the same config is already described [here](https://github.com/thin-edge/thin-edge.io/blob/main/crates/core/tedge/src/bridge/azure.rs#L63).